editor/editor_inspector.cpp

```diff
@@ -3610,8 +3610,8 @@ void EditorInspector::update_tree() {
 				continue;
 			}
 
-			// Hide the "MultiNodeEdit" category for MultiNodeEdit.
-			if (Object::cast_to<MultiNodeEdit>(object) && p.name == "MultiNodeEdit") {
+			// Hide the "MultiNodeEdit" category for MultiNodeEdit and its inherited RefCounted.
+			if (Object::cast_to<MultiNodeEdit>(object) && (p.name == "MultiNodeEdit" || p.name == "RefCounted")) {
```

Comment:
I'd make the condition into a bool variable with descriptive name and possibly extract the logic into a function (also with descriptive name) to get rid of the comment.
Something like:
```
bool shouldNotShow = isMultiNodeEditOrRefCounted(object)
```

Reply:
Don't think that's relevant here, it removes the logic from where it's used which hurts readability